src/core/key.service.ts

```
@@ -510,4 +567,187 @@ export class KeyService {
+   * @param publicKeyPem The PEM-formatted public key
+   * @param keyType The type of the key
+   * @param keyId The key identifier
+   * @returns The JWK representation of the public key
+   */
+  static convertPemToJwk(publicKeyPem: string, keyType: KeyType, keyId: string): JsonWebKey {
+    try {
+      const publicKeyObject = crypto.createPublicKey(publicKeyPem);
+
+      switch (keyType) {
+        case KeyType.RSA: {
+          const keyDetails = publicKeyObject.export({ format: 'jwk' }) as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)